docs/tutorials/gke.md

```diff
@@ -0,0 +1,211 @@
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        image: gcr.io/zalando-docker/external-dns:476c3efddff941ac1b168895c2ba1d5693adc8cc
+        args:
+        - --in-cluster
+        - --zone=external-dns-test-gcp-zalan-do
+        - --source=service
+        - --source=ingress
+        - --dns-provider=google
+        - --google-project=zalando-external-dns-test
+        - --dry-run=false
+```
+
+Create the following sample application to test that `external-dns` works.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: nginx.external-dns-test.gcp.zalan.do.
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 80
+  selector:
+    app: nginx
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        name: nginx
+        ports:
+        - containerPort: 80
+```
+
+After roughly two minutes check that a corresponding DNS record for your service was created.
+
+```console
+$ gcloud dns record-sets list \
+    --zone "external-dns-test-gcp-zalan-do" \
+    --name "nginx.external-dns-test.gcp.zalan.do." \
+    --type A
+NAME                                   TYPE  TTL  DATA
+nginx.external-dns-test.gcp.zalan.do.  A     300  104.155.60.49
+```
+
+Let's check that we can resolve this DNS name. We'll ask the nameservers assigned to your zone first.
+
+```console
+$ dig +short @ns-cloud-e1.googledomains.com. nginx.external-dns-test.gcp.zalan.do.
+104.155.60.49
+```
+
+If you hooked up your DNS zone with it's parent zone correctly you can use `curl` to access your site.
+
+```console
+$ curl nginx.external-dns-test.gcp.zalan.do
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+...
+</head>
+<body>
+...
+</body>
+</html>
+```
+
+Let's check that Ingress works as well. Create the following Ingress.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx
+spec:
+  rules:
+  - host: via-ingress.external-dns-test.gcp.zalan.do
+    http:
+      paths:
+      - backend:
+          serviceName: nginx
+          servicePort: 80
+```
+
+Again, after roughly two minutes check that a corresponding DNS record for your Ingress was created.
+
+```console
+$ gcloud dns record-sets list \
+    --zone "external-dns-test-gcp-zalan-do" \
+    --name "via-ingress.external-dns-test.gcp.zalan.do." \
+    --type A
+NAME                                         TYPE  TTL  DATA
+via-ingress.external-dns-test.gcp.zalan.do.  A     300  130.211.46.224
+```
+
+Let's check that we can resolve this DNS name as well.
+
+```console
+dig +short @ns-cloud-e1.googledomains.com. via-ingress.external-dns-test.gcp.zalan.do.
+130.211.46.224
+```
+
+If you've setup your zone correctly, try with `curl` as well.
```

Comment:
> If you've setup your zone correctly

Same here, instead of questioning the user, assume they did it right, and explain what they need to do in the beginning.